### PR TITLE
Create Lambda create operator and sensor

### PR DESCRIPTION
**Pull request by vincbeck**
Create Lambda create operator and sensor

### CODE REVIEW

File: airflow/providers/amazon/aws/operators/lambda_function.py [hunk 1]
```diff
@@ -20,35 +20,122 @@
...
-    set `invocation_type` to `Event`. For more details,
-    review the boto3 Lambda invoke docs.
+    Creates an AWS Lambda function.
+
+    More information regarding parameters of this operator can be found here
+    https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#Lambda.Client.create_function
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:AwsLambdaCreateFunctionOperator`
```
Comment (Taragolis) [inline airflow/providers/amazon/aws/operators/lambda_function.py]: The only one small thing that I've found. Reference in documentation has a different name

File: docs/apache-airflow-providers-amazon/operators/lambda.rst [hunk 2]
```diff
@@ -33,6 +33,20 @@ Prerequisite Tasks
 Operators
 ---------
 
+.. _howto/operator:AwsLambdaCreateFunctionOperator:
+
+Create an AWS Lambda function
+=============================
+
+To create an AWS lambda function you can use
+:class:`~airflow.providers.amazon.aws.operators.lambda_function.AwsLambdaCreateFunctionOperator`.
```
Comment (Taragolis) [inline docs/apache-airflow-providers-amazon/operators/lambda.rst]: And vice versa

File: airflow/providers/amazon/aws/operators/lambda_function.py (hunk 1 above)
Comment (vincbeck) [reply airflow/providers/amazon/aws/operators/lambda_function.py]: Very nice catch! Thanks :) I just fixed it

**Review by Taragolis — APPROVED** [review]

File: airflow/providers/amazon/aws/operators/lambda_function.py [hunk 3]
```diff
@@ -20,35 +20,122 @@
...
+            self.log.info("Wait for Lambda function to be active")
+            waiter = self.hook.conn.get_waiter("function_active_v2")
+            waiter.wait(
+                FunctionName=self.function_name,
+            )
+
+        return response.get("FunctionArn")
+
+
+class AwsLambdaInvokeFunctionOperator(BaseOperator):
```
Comment (eladkal) [inline airflow/providers/amazon/aws/operators/lambda_function.py:121]: Why the Aws prefix?
This is not aligned with the convention https://github.com/apache/airflow/issues/20296

Comment (vincbeck) [reply airflow/providers/amazon/aws/operators/lambda_function.py:121]: You're totally correct. However, this is not related to this PR, this class existed before this PR, the diff just makes it confusing. I actually dont add `AwsLambdaInvokeFunctionOperator` but `LambdaCreateFunctionOperator`.

Though, I agree with you, this should be fixed and we should rename it to have something consistent. I just created #29677, I propose a bit more than renaming `AwsLambdaInvokeFunctionOperator ` to `LambdaInvokeFunctionOperator`. Let me know what you think (feel free to comment on the issue)

**Pull request merged**